docs/patterns/components/Card/index.css

```diff
@@ -2,7 +2,7 @@
   border-top: 1px solid #ccc;
 }
 
-.CardDemo p {
+.CardDemoText {
```

Comment:
nit: This could be switched to `CardDemo__text` to go along with the BEM syntax we use for the Card classes like `Card__footer` or `Card__header`.

Reply:
Good catch!